playbooks/roles/ims_common/tasks/main.yml

```diff
@@ -71,6 +71,9 @@
           - active_ctl_msg is defined 
           - '"EMERGENCY BACKUP OR OVERRIDE" in active_ctl_msg | join'
 
+      - wait_for:
+          timeout: 10
```

Comment:
Should we add a comment explaining why timeout is needed?

Reply:
ok, will add comment in the next commit